products/content-localization/README.md

```diff
@@ -18,20 +18,110 @@ Based on https://github.com/department-of-veterans-affairs/va.gov-team/issues/53
 # Engineering and Architecture
 
 ## Frontend
-Translations are handled almost entirely within the browser. During page load, [createI18Select()](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/static-pages-entry.js#:~:text=import-,createI18Select,-from%20%27./i18Select/createI18Select) is called to initialize the translation menu links that appear at the top of a translated page. This process includes a check of the current url and looks for a suffix (i.e. `-esp`) indicating the language of the current page, and selects the appropriate language in the translation menu.
+
+### Translation Toggle (English | Español | Tagalog)
+The translation toggles are created in `vets-website`, in a widget referenced by [i18-select](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/static-pages/i18Select).
+
+Translations are handled almost entirely within the browser. During page load, [createI18Select()](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/static-pages-entry.js#:~:text=import-,createI18Select,-from%20%27./i18Select/createI18Select) to create the toggle at the top of a page that has translations. This process includes a check of the current url and looks for a suffix (i.e. `-esp`) indicating the language of the current page, and selects the appropriate language in the translation menu.
```

Comment:
In the second sentence on line 25, I think 'to create' should be changed to something like 'is called to create' or maybe just 'creates'.

Reply:
👍